scripts/ci/lib.sh

```diff
@@ -485,8 +485,6 @@ poll_for_system_test_images() {
 
     local time_limit="$1"
 
-    require_environment "QUAY_RHACS_ENG_BEARER_TOKEN"
```

Comment:
I think this should stay. Most of the images returned by `populate_stackrox_image_list` do require a token.

Reply:
But doesn't this function *fail* in case `QUAY_RHACS_ENG_BEARER_TOKEN` is unset?
If so, we need to remove this guard to enable this code path also for the case of `QUAY_RHACS_ENG_BEARER_TOKEN` being unset.

Reply:
`poll_for_system_test_images` would fail if token is unset, yes. But `poll_for_system_test_images` does not get called in the konflux bundle build context, right? We only need `check_rhacs_eng_image_exists` there, right? Or am I missing something?

Reply:
🤦 Could have sworn I am editing `check_rhacs_eng_image_exists`.